### PR TITLE
Add possibility to use targets in service calls

### PR DESCRIPTION
**Pull request by PiotrMachowski**
This PR adds possibility to use `target` in service calls

### CODE REVIEW

**Comment by JonahKr** [comment]
Hi there @PiotrMachowski 
Could you paste the links to the frontend files to crosscheck?
Thanks alot in advance 👍

**Comment by PiotrMachowski** [comment]
@JonahKr sure!
[types.ts, line 227](https://github.com/home-assistant/frontend/blob/dev/src/types.ts#L227)

File: src/types.ts [hunk 1]
```diff
@@ -28,6 +28,12 @@ export interface CallServiceActionConfig extends BaseActionConfig {
     entity_id?: string | [string];
     [key: string]: any;
   };
+  target?: {
```
Comment (JonahKr) [inline src/types.ts:31]: Is there a reason why you don't reference `HassServiceTarget` as a type from "home-assistant-js-websocket"?

File: src/types.ts [hunk 2]
```diff
@@ -225,7 +231,8 @@ export interface HomeAssistant {
   callService: (
     domain: string,
     service: string,
-    serviceData?: { [key: string]: any }
+    serviceData?: { [key: string]: any },
+    target?: { [key: string]: any }
```
Comment (JonahKr) [inline src/types.ts:235]: Actually the `types.ts` typehints a `HassServiceTarget` as stated above.
if you want you can just add the https://github.com/home-assistant/frontend/blob/95c6adc739a1c632b36f97719e06f5e1f609da37/src/types.ts#L183 
To your PR

File: src/types.ts (hunk 1 above)
Comment (PiotrMachowski) [reply src/types.ts]: The reason is that I was not aware of this type 😃

File: src/types.ts (hunk 2 above)
Comment (PiotrMachowski) [reply src/types.ts]: Thank you, I have added it.

**Pull request merged**